docs/domains/Domain.json

```diff
@@ -573,6 +586,29 @@
         }
       }
     },
+    "MonitoringExporterSpecification": {
+      "type": "object",
+      "properties": {
+        "image": {
+          "description": "The WebLogic Monitoring Exporter sidecar image name. Defaults to ghcr.io/oracle/weblogic-monitoring-exporter:2.0",
+          "type": "string"
+        },
+        "imagePullPolicy": {
```

Comment:
Is it always public open?  Will there be a case that it needs imagePullSecrets?

Reply:
That's a good point. The image would be available publicly available on ghcr.io or OCR. It is possible that the customer would have a custom image for the exporter that is available on a repo that requires image pull secrets.

Reply:
I assume that such a thing is possible, given that the exporter is open source.